markets/perps-market/contracts/interfaces/IAsyncOrderModule.sol

```diff
@@ -32,6 +32,13 @@ interface IAsyncOrderModule {
         address indexed settler
     );
 
+    event OrderCanceled(
+        uint128 indexed marketId,
+        uint128 indexed accountId,
+        uint256 settlementTime,
+        uint256 acceptablePrice
```

Comment:
why do we need the acceptablePrice? (is ok if it was discussed offline, just to know if there's a reason or just because we have the data available)

Reply:
we dont need it necessarily. I added it because it was nice to have